Plugins/Public/minecontrol_plugin/Main.cpp

```diff
@@ -533,30 +352,71 @@ BOOL WINAPI DllMain(HINSTANCE hinstDLL, DWORD fdwReason, LPVOID lpvReserved)
 	// If we're being loaded from the command line while FLHook is running then
 	// set_scCfgFile will not be empty so load the settings as FLHook only
 	// calls load settings on FLHook startup and .rehash.
-	if (fdwReason == DLL_PROCESS_ATTACH && set_scCfgFile.length() > 0)
-		LoadSettings();
+	if (fdwReason == DLL_PROCESS_ATTACH) {
+		if (set_scCfgFile.length() > 0) {
+			LoadSettings();
+		}
+
+		HkLoadStringDLLs();
+	}
+	if (fdwReason == DLL_PROCESS_DETACH)
+	{
+		HkUnloadStringDLLs();
+	}
+	return true;
+
+}
+
+
+bool UserCmd_Process(uint client, const wstring& args)
+{
+	if (args.find(L"/cs") == 0 || args.find(L"/cargostored") == 0)
+	{
+		uint targetId;
+		uint shipId;
+		pub::Player::GetShip(client, shipId);
+		pub::SpaceObj::GetTarget(shipId, targetId);
+		if (!targetId) {
+			PrintUserCmdText(client, L"ERR Mining container not selected");
+			return true;
+		}
+		const auto& container = mapMiningContainers.find(targetId);
+		if (container == mapMiningContainers.end()) {
+			PrintUserCmdText(client, L"ERR Mining container not selected");
+			return true;
+		}
+
+		PrintUserCmdText(client, L"Container holds %u units of cargo", container->second.lootCount);
+	}
 	return true;
 }
 
 void __stdcall PlayerLaunch(unsigned int iShip, unsigned int iClientID)
 {
 	returncode = DEFAULT_RETURNCODE;
-	ClearClientInfo(iClientID);
+	CheckClientSetup(iClientID);
 }
 
 /// Called when a gun hits something
 void __stdcall SPMunitionCollision(struct SSPMunitionCollisionInfo const & ci, unsigned int iClientID)
```

Comment:
Is __stdcall required here

Reply:
The hook is defined as __stdcall, so yes.